ccloud/docs/replicator-to-cloud-configuration-types.rst

```diff
@@ -0,0 +1,211 @@
+.. _replicator-to-cloud-configurations:
+
+|crep-full| to |ccloud| Configurations
+======================================
+
+Whether you are migrating from on-prem to cloud or have a persistent "bridge to cloud" strategy, you can use |crep-full| to copy Kafka data to |ccloud|.
+Learn the different ways to configure |crep| and |kconnect-long|.
+
+.. figure:: images/replicator-to-ccloud.png
+
+===============
+Concepts Review
+===============
+
+Before diving into the different ways to configure |crep|, let's first reprise some basic concepts regarding |crep| and |kconnect-long|.
+This will help you understand the logic for configuring |crep| because how the |kconnect-long| cluster is configured will affect how |crep| should be configured.
```

Comment:
```suggestion
This will help you understand the logic for configuring |crep| because the way that the |kconnect-long| cluster is configured dictates how |crep| should be configured.
```